### PR TITLE
Add a route to expose some metadata

### PR DESCRIPTION
**Pull request by meatballhat**
plus add some tests for `GET /`

### CODE REVIEW

**Comment by meatballhat** [comment]
@igorwwwwwwwwwwwwwwwwwwww r?

**Review by Lyoness — APPROVED** [review]
LGTM :+1:  I could use some more knowledge/education in redis namespaces, but the metadata assertions + test suite look :ok_hand:

**Comment by Lyoness** [comment]
you can add back the label if you want MOAR :eyeglasses: :eyes: on it @meatballhat :heart_decoration:

**Comment by Lyoness** [comment]
also :bicyclist: !!!

**Comment by meatballhat** [comment]
@Lyoness Thank ye!

File: server.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type server struct {
 }
 
 func (srv *server) ohai(w http.ResponseWriter, req *http.Request) {
-	jsonRespond(w, http.StatusOK, &jsonMsg{Message: "ohai™"})
+	jsonRespond(w, http.StatusOK, &jsonMsg{Message: "🚴™"})
```
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline server.go:40]: 🚴

**Pull request merged**